students/Ben_freeman/lesson03/tests/basic_operations_test.py

```diff
@@ -0,0 +1,53 @@
+# pylint: disable=unused-import, unused-wildcard-import,
+# pylint: disable=too-few-public-methods, too-many-arguments, wildcard-import
+# pylint: disable=logging-format-interpolation, pointless-string-statement
+import sys
+import unittest
+sys.path.append("..")
+from src import basic_operations as b_o
+from src import customers
```

Comment:
Change to:

```
import basic_operations as b_o
import customers
```

Reply:
Pylint is still telling me that it cannot load customers or basic_operations even with the changes suggested.
![bluescreenpylint](https://user-images.githubusercontent.com/58352358/77197441-6c279100-6aa2-11ea-8804-b16e7697a24b.PNG)

Reply:
Update the code to your latest and open a new pull request for me to take a look.

-Luis

Reply:
sounds good, thanks!